android/app/build.gradle

```diff
@@ -134,8 +132,10 @@ android {
         applicationId "com.tesseractcollective.communityapp"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 1
-        versionName "1.0"
+        versionCode 2
+        versionName "0.1.1"
+        vectorDrawables.useSupportLibrary = true
+        multiDexEnabled true
```

Comment:
It's super annoying how fast you hit the need for multiDex on react-native apps.